TthCrstl/ReadMe.txt

```diff
@@ -0,0 +1,17 @@
+Tythonian Crystal

```

Comment:
This really should be a "rename" rather than add/delete.
Not sure what's the point of renaming it form README.txt to ReadMe.txt though.

Reply:
Oh.  I replaced all with edited versions and not edit in place. Usually it just marks the differences, but I guess with the name case change, it's fully deleted and added.
README.txt to ReadMe.txt was just for uniformity.

Reply:
Making it an add/delete makes it harder to review, and it makes it harder to follow the history of the file if you want to see what has changed.